### DISCUSSION TITLE
sync_list loaded but ineffective

### DISCUSSION THREAD

**Discussion by everwisher** [abraunegg/onedrive#2083] [Q&A] [answered]
I intend to sync only one folder in my OneDrive. 
#### Docker command script: 
```bash
export ONEDRIVE_UID=`id -u`;
export ONEDRIVE_GID=`id -g`;

docker run -it --name=odtest --restart unless-stopped \
			-e "ONEDRIVE_UID=${ONEDRIVE_UID}" \
			-e "ONEDRIVE_GID=${ONEDRIVE_GID}" \
			-v "/homes/user/od":/onedrive/conf \
			-v "/homes/user/conf":/root/.config/onedrive \
			-v "/data":/onedrive/data \
			driveone/onedrive:latest
```

#### `/homes/user/conf/config`
```bash
space_reservation = "500"
```

#### `/homes/user/conf/sync_list`
```bash
# Exclude *
!*

## Include Test/
Test
```

Using the command `docker exec -it odtest onedrive --display-config`, it returns:
```
Selective sync 'sync_list' configured        = true
sync_list contents:
# Exclude *
!*

## Include Test/
Test
```

However, after authorization, the container still sync all the folders in my OneDrive for Business account.

**Comment by abraunegg** [comment, marked as answer]
@everwisher 
Please re-read https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-sync-via-sync_list-file as your 'Excludes' are not correct - the application by default, when using a 'sync_list' excludes all files ... please correct your configuration.
